ui/lib/sync/addon/components/secrets/page/destinations/destination/details.hbs

```diff
@@ -0,0 +1,38 @@
+<Secrets::Page::Destinations::Destination::Header @destination={{@destination}} />
```

Comment:
I originally had this in the parent `destination.hbs` route template so it rendered for all of the child routes, but that only seems reasonable if we decide to move sync secrets to its own tab (pending design question)

Reply:
I might be missing context but the naming of this component is strange, since it's a header component and not page content 🤔

Reply:
Yeah, it's leftover from when this was in the parent destinations route (so it corresponded to that route template) 

I can rename